core/src/main/java/nl/nn/adapterframework/doc/ModelBuilder.java

```diff
@@ -0,0 +1,43 @@
+package nl.nn.adapterframework.doc;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import lombok.Getter;
+import nl.nn.adapterframework.doc.model.FrankDocModel;
+import nl.nn.adapterframework.doc.objects.SpringBean;
+
+public class ModelBuilder {
+	private @Getter FrankDocModel model;
+
+	public ModelBuilder() {
+		model = new FrankDocModel();
+	}
+
+	/**
+	 * @param interfaceName The interface for which we want SpringBean objects.
+	 * @return All classes implementing interfaceName, ordered by their full class name.
+	 */
+	static List<SpringBean> getSpringBeans(final String interfaceName) {
```

Comment:
public/private ?

Reply:
Niet private, want dan kan ik hem niet unit testen. Ik denk dat ik in deze class uiteindelijk het productie FrankDocModel ga genereren, dus hier komt een method waarin alle groepen worden gemaakt en waarin alle FrankElement objecten worden gemaakt. Daarvoor heb ik waarschijnlijk een nieuwe method nodig die getSpringBeans() gaat gebruiken. Die nieuwe method wordt dan public, dus getSpringBeans() hoeft niet public te worden.